feature-libs/cart/main/components/cart/added-to-cart-dialog/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './added-to-cart-dialog.component';
+export * from './added-to-cart-dialog.module';
```

Comment:
I have no strong opinion on that topic. Since we export the rest of the UI code, I didn't see a reason not to export the listener.  
The checkout lib exports it's listener ( although it is in `core`, not `components`  )

Do you think it should be private instead?

Reply:
I think we can export the listener. `onAddToCart` and `openModal` both are `protected`.

Reply:
Yes.